dlp/quickstart.py

```diff
@@ -22,20 +22,31 @@ def quickstart():
     """Demonstrates use of the Data Loss Prevention API client library."""
 
     # [START quickstart]
+    import os
+    import sys
+
     # Import the client library
-    import google.cloud.dlp_v2beta1
+    import google.cloud.dlp
 
     # Instantiate a client.
-    dlp = google.cloud.dlp_v2beta1.DlpServiceClient()
+    dlp = google.cloud.dlp.DlpServiceClient()
+
+    # Pull the project ID from an environment variable.
```

Comment:
Eh, don't do this. I would just prefer you to do:

```
project = 'YOUR_PROJECT'
```

Reply:
Done